frontend/test/acceptance/pages/EnterDetails.scala

```diff
@@ -47,20 +51,37 @@ case class EnterDetails(val testUser: TestUser) extends Page with Browser {
     def selectCountryCode(countryCode:  String) { setSingleSelectionValue(country, countryCode) }
   }
 
-  private object CreditCard {
-    val cardNumber = id("cc-num")
-    val cardCvc = id("cc-cvc")
-    val cardExpiryMonth = id("cc-exp-month")
-    val cardExpiryYear = id("cc-exp-year")
-
-    private def fillInHelper(cardNum: String) {
-      setValue(cardNumber, cardNum)
-      setSingleSelectionValue(cardExpiryMonth, "10")
-      setSingleSelectionValue(cardExpiryYear, "2019")
-      setValue(cardCvc, "111")
+  // Temporary hack to identify elements on Stripe Checkout form using xpath, since the ids are no longer consistently set.
+  private object StripeCheckout {
+    val container = name("stripe_checkout_app")
+    val cardNumber = xpath("//div[label/text() = \"Card number\"]/input")
+    val cardExp = xpath("//div[label/text() = \"Expiry\"]/input")
+    val cardCvc = xpath("//div[label/text() = \"CVC\"]/input")
+    val submitButton = xpath("//div[button]")
+
+    private def fillInHelper(cardNum: String) = {
+
+      setValueSlowly(cardNumber, cardNum)
+      setValueSlowly(cardExp, "1019")
+      setValueSlowly(cardCvc, "111")
+      continue()
+      Thread.sleep(5000)
```

Comment:
I wield no hammer so this is just an opinion: If you need to use `Thread.sleep` to make the test work, you should be rethinking the test (but I know this is a hack taken from subs-fe)